docker-aliases.sh

```diff
@@ -17,7 +17,7 @@ fi
 alias ede='unset $(bash docker/dotenv-vars.sh) && export $(php docker/export-dotenv-vars/app.php $(bash docker/dotenv-vars.sh))'
 # dcu = docker(d) compose(c) up(u)
 PRE_DCU="bash set_symfony_version.sh"
-DCU_BASE="MSYS_NO_PATHCONV=1 BLACKFIRE_SERVER_ID=${BLACKFIRE_SERVER_ID} BLACKFIRE_SERVER_TOKEN=${BLACKFIRE_SERVER_TOKEN} BLACKFIRE_CLIENT_ID=${BLACKFIRE_CLIENT_ID} BLACKFIRE_CLIENT_TOKEN=${BLACKFIRE_CLIENT_TOKEN} CADDY_MERCURE_JWT_SECRET=${CADDY_MERCURE_JWT_SECRET} docker compose"
+DCU_BASE="MSYS_NO_PATHCONV=1 BLACKFIRE_APM_ENABLED="${BLACKFIRE_APM_ENABLED:-0}" BLACKFIRE_SERVER_ID=${BLACKFIRE_SERVER_ID} BLACKFIRE_SERVER_TOKEN=${BLACKFIRE_SERVER_TOKEN} BLACKFIRE_CLIENT_ID=${BLACKFIRE_CLIENT_ID} BLACKFIRE_CLIENT_TOKEN=${BLACKFIRE_CLIENT_TOKEN} CADDY_MERCURE_JWT_SECRET=${CADDY_MERCURE_JWT_SECRET} docker compose"
```

Comment:
so you can control if blackfire apm is enabled using the environment variable BLACKFIRE_APM_ENABLED. You can just put it in ~/.bashrc. I updated the https://confluence.cradle.buas.nl/display/MDD/Server+Development+Setup page